services/frontend-service/src/ui/components/ServiceLane/EnvSelectionDialog.tsx

```diff
@@ -56,8 +55,12 @@ export const EnvSelectionDialog: React.FC<EnvSelectionDialogProps> = (props) =>
     );
 
     return (
-        <Dialog open={props.open}>
-            <DialogTitle id="alert-dialog-title">{'Select all environments to be removed:'}</DialogTitle>
+        <ConfirmationDialog
+            onConfirm={onConfirm}
+            onCancel={onCancel}
+            open={props.open}
+            headerLabel={'Select all environments to be removed:'}
+            confirmLabel={'Remove environments from app'}>
```

Comment:
I think `Remove application from environments` is the correct action.

Reply:
done